### PR TITLE
Added C# example for NVDA controller client DLL

### PR DESCRIPTION
**Pull request by DataTriny**
### Link to issue number:

Based on #7010.

### Summary of the issue:

The original author of #7010 seems not to be active anymore, even if there are still review actions to perform on the PR.

### Description of how this pull request fixes the issue:

This PR introduces a C# program containing a PInvoke wrapper around nvdaControllerClient.dll, and demonstrates how to use it.
I also updated the readme. I think that the link pointing to the pre-built binaries should be updated as well.

### Testing performed:

Performed the same tasks as in the Python example.

### Known issues with pull request:

None.

### Change log entry:

Section: New features
- Added a new example to demonstrate how to call the nvdaControllerClient.dll from C#.

### CODE REVIEW

File: extras/controllerClient/x86/example.cs [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+			nvdaController_speakText(text);
+		}
+	}
+
+	class Program
+	{
+		static void Main(string[] args)
+		{
+			// Test if NVDA is running.
+			if (!NVDA.IsRunning)
```
Comment (feerrenrut) [inline extras/controllerClient/x86/example.cs]: Please add braces around the scope of the if block.

**Review by feerrenrut — CHANGES_REQUESTED** [review]
Thanks for taking this on.
It looks like the file is included twice. Please delete the `example.cs` one, the other comments for that file also apply to the `example_csharp.cs` file

**Comment by DataTriny** [comment]
Sorry, but it is so common in the .NET world to adopt this coding style.
Everything fixed!

**Comment by feerrenrut** [comment]
No problem, I understand the temptation. I have just seen too much wasted time because of it.  Thanks for making the changes.

**Review by feerrenrut — APPROVED** [review]

**Pull request merged**